.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,7 +1,8 @@
-Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issues #IssueNumber]
+Fixes #[Add issue number here. If you do not solve the issue entirely, please change the message e.g. "First steps for issue" #IssueNumber]
 
-Changes: [Add here what changes were made in this issue and if possible provide links.]
+Changes: [Add here what changes you made in this issue and if possible provide links referring the changes.]
 
 Surge Deployment Link: https://pr-${TRAVIS_PULL_REQUEST}-fossasia-susi-accounts.surge.sh
 
-Screenshots for the change:
+Screenshots for the change: [Add here screenshots depicting the changes you made in order to resolve this issue.]
```

Comment:
I think the `Screenshots for the change` explains itself, no need to explain it more.

Reply:
@neeraj3029 Thanks for review
I've made the changes as suugested.